src/platform/linux/kmsgrab.cpp

```diff
@@ -1647,8 +1660,9 @@ namespace platf {
         }
 
         if (!fb->handles[0]) {
-          BOOST_LOG(error)
-            << "Couldn't get handle for DRM Framebuffer ["sv << plane->fb_id << "]: Possibly not permitted: do [sudo setcap cap_sys_admin+p sunshine]"sv;
+          BOOST_LOG(error) << "Couldn't get handle for DRM Framebuffer ["sv << plane->fb_id << "]: Probably not permitted"sv;
+          BOOST_LOG((window_system != window_system_e::X11 || config::video.capture == "kms") ? fatal : error)
+            << "You must run [sudo setcap cap_sys_admin+p $(readlink -f sunshine)] for KMS display capture to work!"sv;
```

Comment:
Should we include a link to this `https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/setup.html#install` for more info, or the event they are using the flatpak/appimage?

There's also a cmake var "SUNSHINE_BUILD_FLATPAK" that could be exposed as a compile definition, if you want to provide a different message for flatpak. And "SUNSHINE_BUILD_APPIMAGE" for AppImage.

Reply:
Leaving this as-is per our discussion on Discord that the link doesn't take you directly to where you need to read.